tools/gromacs/sim.xml

```diff
@@ -43,15 +43,15 @@
             -p ./top_input.top 
             -o outp.tpr &>> verbose.txt &&
 
-
-        gmx mdrun
-            -nt "\${GALAXY_SLOTS:-4}"
-            ## GPU
-            #if "\${GROMACS_GPU:-0}" == 1:
+        #if "\${GPU_AVAILABLE:-0}" == 1:
```

Comment:
I don't know if a variable like `$GPU_AVAILABLE` exists, I hope it does.

If not, I think something like this would work:
```
if [ `nvcc --version | grep -q 'NVIDIA' && echo $? == 0` ]; then
    gmx_gpu mdrun ... ;
else
    gmx mdrun ...; fi
```
But I've never seen bash if statements in Galaxy tools, so I assume it's discouraged.

Reply:
So what is the best option here? There is this `GPU_ENABLED` included in https://github.com/galaxyproject/galaxy/pull/10127 but I imagine this will take a while to get merged.